src/sys/DISK.ASM

```diff
@@ -973,7 +973,7 @@ check_fname_legal:                              ; Check a character for filename
 
 
 ; NEXT AVAILABLE CLUSTER: Searches the FAT for unused entries, grabs the first unused cluster number
-; IN: None
+; IN: [cluster] = Exception cluster
```

Comment:
Added this check because the same cluster would be written to over and over again.